src/index.ts

```diff
@@ -30,40 +30,104 @@
  * },
  */
 
-/**
- * @typedef {object} ImageToolData
- * @description Image Tool's input and output data format
- * @property {string} caption — image caption
- * @property {boolean} withBorder - should image be rendered with border
- * @property {boolean} withBackground - should image be rendered with background
- * @property {boolean} stretched - should image be stretched to full width of container
- * @property {object} file — Image file data returned from backend
- * @property {string} file.url — image URL
- */
-
+import type { TunesMenuConfig } from "@editorjs/editorjs/types/tools";
+import type { API, ToolboxConfig, PasteConfig, BaseTool } from '@editorjs/editorjs';
 import './index.css';
 
 import Ui from './ui';
 import Uploader from './uploader';
 
 import { IconAddBorder, IconStretch, IconAddBackground, IconPicture } from '@codexteam/icons';
+import { TunesConfig } from './types/types';
+
+/**
+ * ImageToolData interface representing the input and output data format for the image tool.
+ */
+export interface ImageToolData {
```

Comment:
why some types are moved to the `types.ts` but others left here? You need to decide where there should be